recipes/slang/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "slang" %}
```

Comment:
Given that this is in another pr, can we remove this?

Reply:
@mariusvniekerk, this is now done and ready to merge.